src/skmultiflow/trees/nodes/active_learning_node_perceptron.py

```diff
@@ -1,7 +1,5 @@
-from copy import deepcopy
-
 import numpy as np
-
+from copy import deepcopy
```

Comment:
Original structure follows PEP8 guidelines: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
You are right! My bad, I'll revert that. Thanks for pointing that you.

Reply:
I reverted the change and enforced PEP8's import style to the other regression learning nodes in commit [f1cef7e](https://github.com/scikit-multiflow/scikit-multiflow/pull/230/commits/f1cef7331233cbc05754b1495b97b64893019f50).